packages/frontend/core/src/modules/workbench/entities/workbench.ts

```diff
@@ -47,21 +47,50 @@ export class Workbench extends Entity {
     return views.at(activeIndex) || views[0];
   });
 
+  resizingView$ = new LiveData<{
```

Comment:
I think these states should not be here. These should be UI states. Can you organize them into a component?

Reply:
I thought these states are shared between multiple components and feel like passing them around is too messy. Turns out they are only used in a single component in the latest impl